api/src/main/java/org/openmrs/util/PrivilegeConstants.java

```diff
@@ -568,5 +568,8 @@ private PrivilegeConstants() {
 
 	@AddOnStartup(description = "Able to delete diagnoses")
 	public static final String DELETE_DIAGNOSES = "Delete Diagnoses";
+	
+    @AddOnStartup(description = "Able to create a provider by User")
+	public static final String CREATE_PROVIDERS = "Create Diagnoses by User";
```

Comment:
Do we need a new privilege here?

Reply:
Thanks @ibacher, unfortunately i didnot see that save privilege being used in this class so that was my first intension, let me remove it

Reply:
Since earlier before you indicated that we have create_Providers by user which i had not recognised, this should be got rid of thanks